src/main/java/com/puppycrawl/tools/checkstyle/Main.java

```diff
@@ -724,8 +724,8 @@ private static final class CliOptions {
         /** Option that controls whether to print the AST of the file. */
         @Option(names = {"-t", "--tree"},
                 description = "Prints Abstract Syntax Tree(AST) of the checked file. The option "
-                        + "cannot be used other options and requires exactly one file to run on "
-                        + "to be specified.")
+                        + "cannot be used with other options and requires exactly one file to "
+                        + "run on to be specified with the complete path.")
```

Comment:
Do you mean relative path is not working?
Can you share cli example?

Reply:
> Do you mean relative path is not working?

No, It works obviously..

I didn't mean to say just absolute path.. 
I was trying to find the right words which indicate both "absolute" & "relative" paths 😕 
```suggestion
                        + "run on to be specified with the path pointing to this file.")
```

Is this above wording clear on my intention ?

Because, " `one file to run on to be specified` " made many new contributors use just the filename like 
" `TestXxxFile.java` " which gives missing file error as we know and yep now that you mentioned, even "complete path" sounds misleading

Reply:
It is just `path` is is clear that it must be valid. Users of CLI has to understand concept of `current folder`.